src/test/java/marquez/api/JobResourceTest.java

```diff
@@ -333,12 +333,12 @@ private Response getJob(String jobName) {
   }
 
   private Response insertJob(Job job) {
-    CreateJobRequest createJobRequest =
-        new CreateJobRequest(
-            job.getLocation(),
-            job.getDescription(),
+    JobRequest createJobRequest =
```

Comment:
maybe renaming 'createJobRequest' to 'jobRequest' for consistency

Reply:
Good catch!